src/content/en/2021/performance.md

```diff
@@ -1,20 +1,532 @@
+  caption="FID performance by connection type",
+  description="Stacked bar chart showing FID performance for effective connection type. For offline websites, 78% had good, 17% needs improvement, and 5% poor performance. For slow 2G and 2G, 81-82% were good, 18% needs improvement, and 0-1% poor. For 3G, 90% were good and 10% needs improvement. For 4G, 93% were good, 7% needs improvement, and 0% poor.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQj6-t49BNV_5W-w83AABkUoo6klfyUyIz13yKShLAzK8qGs5lJ9TKcggIEp6JgxikVF-UJBAHpsrNl/pubchart?oid=1135701788&format=interactive",
+  sheets_gid="730149797",
+  sql_file="web_vitals_by_eff_connection_type.sql"
+  )
+}}
+
+FID performance degraded some by connection type, but less so than the other metrics. The high distribution of scores seems to be reducing the amount of variance we're seeing in the analysis.
+
+Unlike the other metrics, FID was worse for offline websites than any other connection category. This could be due to the more complex nature of many websites with service workers. Having a service worker does not eliminate the impact of client-side JavaScript running on the main thread.
+
+{{ figure_markup(
+  image="performance-FID-by-rank.png",
+  caption="FID performance by rank",
+  description="Stacked bar chart showing FID performance for each ranking group. For all categories, 93-94% had good, 5-7% needs improvement, and 0% poor performance.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQj6-t49BNV_5W-w83AABkUoo6klfyUyIz13yKShLAzK8qGs5lJ9TKcggIEp6JgxikVF-UJBAHpsrNl/pubchart?oid=1539782601&format=interactive",
+  sheets_gid="730149797",
+  sql_file="web_vitals_by_rank.sql"
+  )
+}}
+
+FID performance by rank was flat.
+
+For all FID metrics, we see very large bars in the "good" category which makes it less effective unless we've truly hit peak performance. The good news is the <a hreflang="en" href="https://web.dev/better-responsiveness-metric/">Chrome team is evaluating this now</a> and would like your feedback.
+
+If your site's performance is not in the "good" category, then you definitely have a performance problem. A common culprit for FID issues is too much long-running JavaScript. Keep your bundle sizes small and pay attention to third-party scripts.
+
+### Total Blocking Time (TBT)
+
+> The Total Blocking Time (TBT) metric measures the total amount of time between First Contentful Paint (FCP) and Time to Interactive (TTI) where the main thread was blocked for long enough to prevent input responsiveness. --<a hreflang="en" href="https://web.dev/tbt/">Web.dev</a> ]
+
+<a hreflang="en" href="https://web.dev/tbt/">Total Blocking Time (TBT)</a> is a lab-based metric that helps us debug potential interactivity issues. FID is a field-based metric, and TBT is its lab-based analog. Currently, when evaluating client websites, I reach for total blocking time TBT as another indicator of possible performance issues due to JavaScript.
```

Comment:
> I reach

to

> you can look at

or rewrite without first person

Reply:
This was a case where I thought it was too personal-opinion of me so wanted to clarify not necessarily the Web Almanac view

Reply:
I am just not sure the reader makes the distinction between you and the Web Almanac.  If this information was unreliable they're more likely to think it's the Web Almanac irregardless of how you voice it.  Just, again, jarring to go from data driven text to first person.

Reply:
See guidance here https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#general-principles-for-technical-writing